common/src/Microsoft.Azure.IIoT.Hub.Module.Framework/src/Hosting/ModuleHost.cs

```diff
@@ -91,6 +91,9 @@ public sealed class ModuleHost : IModuleHost, ITwinProperties, IEventEmitter,
                     _logger.Error(ce, "Module Host stopping caused exception.");
                 }
                 finally {
+                    kModuleStart.WithLabels(DeviceId ?? "", ModuleId ?? "", ModuleGuid, "",
+                        DateTime.UtcNow.ToString("yyyy-MM-dd'T'HH:mm:ss.FFFFFFFK", 
+                        CultureInfo.InvariantCulture)).Set(0);
                     kModuleStart.WithLabels(DeviceId ?? "", ModuleId ?? "",
                         DateTime.UtcNow.ToString("yyyy-MM-dd'T'HH:mm:ss.FFFFFFFK",
                         CultureInfo.InvariantCulture)).Set(0);
```

Comment:
why is duplicate? btw the guid is missing causing probably   an exception.
Perhaps we should always guard setting the metrics with labels in try ... catch blocks

Reply:
missed it